src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/WpfBasedPropertyPage.cs

```diff
@@ -44,7 +44,9 @@ protected override async Task OnSetObjectsAsync(bool isClosing)
 
         protected override Task<int> OnApplyAsync()
         {
+#pragma warning disable VSTHRD110 // Observe result of async calls
```

Comment:
Do we know why this is necessary? On the surface it looks like a bug in the analyzer.

Reply:
In this scenario I am not sure why this is needed.

Should suppress it at a higher level ?

Reply:
No, this is the right place to suppress it, but it looks like there's a bug in the analyzer that prevents it from properly understanding this code pattern.

I suggest filing a bug at https://github.com/microsoft/vs-threading, and including the link to the bug with the suppression.

Reply:
@sharwell

Reply:
yes it's a bug

Reply:
https://github.com/microsoft/vs-threading/issues/880